### PR TITLE
feat: allow to pass extra parameters for celery workers

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR allows to pass custom parameters to any celery worker deployment

### CODE REVIEW

File: tutorcelery/patches/k8s-deployments [hunk 1]
```diff
@@ -31,6 +31,9 @@ spec:
             - "--hostname=edx.{{service}}.core.{{variant}}.%%h"
             - "--max-tasks-per-child=100"
             - "--queues=edx.{{service}}.core.{{variant}}"
+            {% for setting, value in config.get("extra_params", {}).items() %}
+            - "--{{setting}}={{value}}"
+            {% endfor %}
```
Comment (MoisesGSalas) [inline tutorcelery/patches/k8s-deployments]: Maybe we can have the logic for the command in the python code and use a template filter to render the final command.

Comment (marbonilla) [inline tutorcelery/patches/k8s-deployments]: could this be susceptible to malicious injection of some sort?

Comment (Ian2012) [reply tutorcelery/patches/k8s-deployments]: I don't think so

Comment (Ian2012) [reply tutorcelery/patches/k8s-deployments]: I prefer rendering this code on the template, it's just a setting and a value

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**